Storage/ThirdParty/SwiftData.swift

```diff
@@ -104,49 +134,36 @@ open class SwiftData {
         assert(sqlite3_threadsafe() == 2)
     }
 
-    fileprivate func getSharedConnection() -> ConcreteSQLiteDBConnection? {
```

Comment:
Is this ok to remove? I thought it was necessary to make this a `sync` operation on the queue to prevent races?

Reply:
Being a serial queue I am not sure it can race.